x-pack/plugins/fleet/common/settings/agent_policy_settings.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+import { i18n } from '@kbn/i18n';
+
+import type { SettingsConfig } from './types';
+
+export const AGENT_POLICY_ADVANCED_SETTINGS: SettingsConfig[] = [
+  {
+    name: 'agent.limits.go_max_procs',
+    saved_object_field: {
+      name: 'agent_limits_go_max_procs',
+      mapping: {
+        type: 'integer',
+        index: false,
+      },
+    },
+    title: i18n.translate('xpack.fleet.settings.agentPolicyAdvance.goMaxProcsDescription', {
```

Comment:
the `i18n.translate` part could be added by default on the dynamic component title and description fields with a generated key, to avoid repeating it

Reply:
moved i18n to the form settings component

Reply:
hm it seems i18n doesn't allow a generated id, it has to be a string literal:
```
ERROR  I18N ERROR  Error in x-pack/plugins/fleet/public/applications/fleet/components/form_settings/index.tsx
      Error: Message id should be a string literal.
```

Reply:
reverted the change, I don't see a better solution now than keeping the i18n in the config